package.json

```diff
@@ -20,36 +20,42 @@
       "description": "Command or full path to pylint. Use %p for current project directory (no trailing /) or %h for current project name.",
       "order": 1
     },
+    "additionalArgs": {
```

Comment:
This is automatically translated into a title, currently this would be "Additional Args". Can you change this to `additionalArguments` ("Additional Arguments")?